bounties/rubygems/webbynode/1/README.md

```diff
@@ -0,0 +1,3 @@
+# Overview
+
+`webbynode` is a Webbynode Deployment Gem. This Gem contains a vulnerability in `notify.rb` that is triggered when handling a specially crafted `growlnotify` message. This can allow a context-dependent attacker to execute arbitrary commands.
```

Comment:
The issue exists on https://github.com/webbynode/webbynode/blob/master/lib/webbynode/notify.rb#L10-L11.

The `regex` check doesn't prevent the attack, so the issue is `valid` :smile:

![Screenshot from 2020-08-05 11-02-38](https://user-images.githubusercontent.com/33063403/89393669-6b1e9580-d70b-11ea-986d-5517dba1fcb9.png)